docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_navigation:
+  title: Share work packages
+  priority: 963
+description: How to share work packages in OpenProject.
+keywords: work package, share, share work packages
+---
+
+# Share work packages
+
+Since the 13.1 it is possible to share work packages with users that are not members of a project or are not yet registered on your instance. In the latter case a user will need to register in order to view the work package.
+
+> **Note**: Sharing work packages with non-member is an Enterprise add-on and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or  [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free community edition is easy and helps support OpenProject.
+
+To share a work package with a project non-member select the detailed view of a work package and click the **Share** button.
+
+![Share button in OpenProject work packages](openproject_user_guide_share_button_wp.png)
+
+A dialogue window will open, showing the list of all users, who this work package has already been shared with. If the work package has not yet been shared, the list will empty. 
+
+> **Note**: For sharing with non members to be available, a respective role needs to be assigned to the **Non member role**. 
```

Comment:
I'm not sure what this phrase wants to say. Is this about giving a share role to the external user?